module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -549,11 +549,21 @@ public function getHolding($bibId, array $patron = null, array $options = [])
                 );
                 $locationId = $item->effectiveLocationId;
                 $locationName = $this->getLocationName($locationId);
+                // concatenate enumeration fields if present
+                $enum = join(
+                    ' ', array_filter(
+                        [
+                            $item->volume ?? null,
+                            $item->enumeration ?? null,
+                            $item->chronology ?? null]
+                    )
+                );
                 $items[] = [
                     'id' => $bibId,
                     'item_id' => $item->id,
                     'holding_id' => $holding->id,
                     'number' => count($items) + 1,
+                    'enumchron' => $enum ?? '',
```

Comment:
I'm pretty sure the `?? ''` is unnecessary here, since $enum should never be null -- `implode` will always return a string.